src/test/java/org/sagebionetworks/web/unitclient/presenter/SearchPresenterTest.java

```diff
@@ -256,4 +557,14 @@ public void testEncodedQueryTerm() throws Exception {
     verify(mockJsClient)
       .getSearchResults(eq(expectedQuery), any(AsyncCallback.class));
   }
+
+  @Test
+  public void testGetSearchResultClickedHandler() throws Exception {
```

Comment:
?

Reply:
You tested the click handler above, should this test be deleted?

Reply:
Yes it should, thanks